packages/react-components/theme-designer/src/components/Demo/Demo.tsx

```diff
@@ -0,0 +1,222 @@
+  const CutIcon = bundleIcon(CutFilled, CutRegular);
+  const PasteIcon = bundleIcon(ClipboardPasteFilled, ClipboardPasteRegular);
+  const EditIcon = bundleIcon(EditFilled, EditRegular);
+  return (
+    <Menu>
+      <MenuTrigger>
+        <MenuButton>Select </MenuButton>
+      </MenuTrigger>
+      <MenuPopover>
+        <MenuList>
+          <MenuItemCheckbox icon={<CutIcon />} name="edit" value="cut">
+            Cut
+          </MenuItemCheckbox>
+          <MenuItemCheckbox icon={<PasteIcon />} name="edit" value="paste">
+            Paste
+          </MenuItemCheckbox>
+          <MenuItemCheckbox icon={<EditIcon />} name="edit" value="edit">
+            Edit
+          </MenuItemCheckbox>
+        </MenuList>
+      </MenuPopover>
+    </Menu>
+  );
+};
+
+export const Column2 = () => {
+  const styles = useStyles();
+  return (
+    <div className={styles.col2}>
+      <TabList defaultSelectedValue="tab1">
+        <Tab value="tab1">Home</Tab>
+        <Tab value="tab2">Pages</Tab>
+        <Tab value="tab3">Documents</Tab>
+      </TabList>
+      <Input
+        placeholder="Find"
+        contentAfter={<Button aria-label="Find" appearance="transparent" icon={<SearchRegular />} size="small" />}
+      />
+      <DemoMenu />
+    </div>
+  );
+};
+
+export const DemoIcons = () => {
+  const styles = useStyles();
+  const MeetNowIcon = bundleIcon(MeetNowFilled, MeetNowRegular);
+  const CalendarLtrIcon = bundleIcon(CalendarLtrFilled, CalendarLtrRegular);
+  return (
+    <div className={styles.icons}>
+      <Badge size="medium" appearance="filled" icon={<CalendarLtrIcon />} />
+      <Badge size="medium" appearance="ghost" icon={<CalendarLtrIcon />} />
+      <Badge size="medium" appearance="outline" icon={<MeetNowIcon />} />
+      <Badge size="medium" appearance="tint" icon={<MeetNowIcon />} />
+    </div>
+  );
+};
+
+export const Column3 = () => {
+  const styles = useStyles();
+  return (
+    <div className={styles.col3}>
+      <div className={styles.button}>
+        <Button appearance="primary">Sign Up</Button>
+      </div>
+      <div className={styles.button}>
+        <Button appearance="transparent" icon={<ChevronRightRegular />} iconPosition="after">
+          Learn More
+        </Button>
+      </div>
+      <Slider className={styles.slider} defaultValue={20} />
+      <DemoIcons />
+      <div className={styles.twoRow}>
+        <Switch defaultChecked={true} label="On" />
+        <Switch label="Off" />
+      </div>
+      <div className={styles.twoRow}>
+        <Checkbox defaultChecked={true} label="Option 1" />
+        <Checkbox label="Option 2" />
+      </div>
+      <div className={styles.twoRow}>
+        <Radio defaultChecked={true} label="Option 1" />
+        <Radio label="Option 2" />
+      </div>
+    </div>
+  );
+};
+
+export const Demo: React.FC<ContentProps> = props => {
+  const styles = useStyles();
+  return (
+    <FluentProvider theme={teamsLightTheme}>
+      <Label>Examples</Label>
```

Comment:
Label's are typically for inputs. Just use regular text here. And this text should be inside of the root